src/Symfony/Component/Debug/Debug.php

```diff
@@ -28,10 +32,11 @@ class Debug
      * If the Symfony ClassLoader component is available, a special
      * class loader is also registered.
      *
-     * @param int  $errorReportingLevel The level of error reporting you want
-     * @param bool $displayErrors       Whether to display errors (for development) or just log them (for production)
+     * @param int                $errorReportingLevel The level of error reporting you want
+     * @param bool               $displayErrors       Whether to display errors (for development) or just log them (for production)
+     * @param FormatterInterface $formatter           The output formatter.
      */
-    public static function enable($errorReportingLevel = E_ALL, $displayErrors = true)
+    public static function enable($errorReportingLevel = E_ALL, $displayErrors = true, FormatterInterface $formatter = null)
```

Comment:
I don't think we need add this argument: this is a helper, let's keep it simple, if people want to inject their formatter, they can do the job by themselves.